src/components/swimlanes/swimLanes.tsx

```diff
@@ -78,13 +81,7 @@ const SwimLanes = ({ lanesWithNumItems }) => {
                 <Text
                   id={`item-count-${lane.slug}-${index}`}
                   size="subtitle2"
-                  sx={{
-                    fontWeight: "400",
-                    display: "none",
-                    [`@media screen and (min-width: 600px)`]: {
-                      display: "inline",
-                    },
-                  }}
```

Comment:
This is needed because it hides this text on mobile and displays it on desktop. If this was intentional then ok, I just don't see where this was stated as an update.

For the font-weight, however, it was initially set as "400" so you could update it to `fontWeight: "regular"`.